source/pkx/PK4.cpp

```diff
@@ -850,6 +852,134 @@ void PK4::partyLevel(u8 v)
     }
 }
 
+std::shared_ptr<PKX> PK4::convertToG3(Sav& save) const
+{
+    // TODO: Not sure how to do ribbons or if trash bytes are needed
```

Comment:
Trash bytes are not needed. Ribbons will need to be mapped manually

Reply:
I removed the commented trash bytes stuff and moved the TODO more directly to the ribbons for now